reference/src/representation/function-pointers.md

```diff
@@ -8,7 +8,16 @@ In Rust, a function pointer type, is either `fn(Args...) -> Ret`,
 A function pointer is the address of a function,
 and has function pointer type.
 The pointer is implicit in the `fn` type,
-and `fn` types are implicitly `'static`.
+and they have no lifetime of their own;
+therefore, function pointers are assumed to point to
+a block of code with static lifetime.
+This is not necessarily always true,
+since, for example, you can unload a dynamic library.
+Therefore, this is _only_ a safety invariant,
+not a validity invariant;
+as long as one doesn't call a function pointer which points to freed memory,
+it is not undefined behavior.
```

Comment:
I'd just say something like: "Function pointers have no lifetime. They must point to an allocated block of code when being called."

The rest of this belongs to https://github.com/rust-rfcs/unsafe-code-guidelines/issues/72, doesn't it?